### DISCUSSION TITLE
Create Streamyard open source alternative

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#3174] [Ideas]
**Features v0.1**
- single user, no auth
- merge camera, screen, and guests

**Features v0.2**
- oauth login
- output to YouTube and Twitch

**Comment by eddiejaoude** [comment]
What open source libraries could help with this? 👇

**Reply by suitupstan** [reply]
Maybe, https://github.com/webrtcHacks/adapter

**Reply by eddiejaoude** [reply]
Merging live streams https://www.npmjs.com/package/video-stream-merger

**Comment by eddiejaoude** [comment]
Is this a good idea? 👇

**Reply by suitupstan** [reply]
Maybe 🤷 IMHO A lot of work to do, while our other projects are still in dev.

**Reply by eddiejaoude** [reply]
yes you are right! 👍

**Comment by Pradumnasaraf** [comment]
### Those who are wondering,  what StreamYard is? check out [this](https://streamyard.com/teams/jiRCXQWoVC8E7sAkSFataJPi/broadcasts)

**Reply by suitupstan** [reply]
Also check out https://alternativeto.net/software/streamyard/

**Comment by eddiejaoude** [comment]
We have a tiny demo of merging multiple input streams into a single output stream for YouTube or other platforms 

https://user-images.githubusercontent.com/624760/147344471-d04adc98-f296-4297-8275-488e79e13785.mov

**Comment by Cahllagerfeld** [comment]
Due to the complexity I'd definitely use TypeScript for the Project

**Comment by weismannweb** [comment]
Would you consider making a contribution to jitsi for this?  I would love to see https://jitsi.org/ have the ability to push to multiple rtmp.  It already supports pushing to one...but i dont think multiple.  Also the unified chat is nice on restream and streamyard.

**Comment by dfernandesbsolus** [comment]
Do you have any updates on this? I have some ideas on how to do this.

**Comment by HeraclesG** [comment]
Typescript is the best I think.
